src/base/misc/fatfs.c

```diff
@@ -1577,7 +1577,8 @@ void build_boot_blk(fatfs_t *f, unsigned char *b)
     case OLDMSD_D:
       txfr->ofs = 0x0000;
       txfr->seg = 0x0070;
-      txfr->ax  = d_o >> 16;
+      txfr->ax  = 0x0200;  /* Magic value which makes MS-DOS 3.10 happy, see */
```

Comment:
Yep agreed, but that's the value of AX at the start of boot, and as I said I don't understand what it is.